hconfig_utils/hconfig_r4.F90

```diff
@@ -0,0 +1,10 @@
+module hconfig_r4
```

Comment:
I am thinking that we need a mapl prefix for even this semi-hidden modules.   But feel free to explain a rationale for otherwise.

Reply:
will do